discussment-learning/src/main/java/org/danekja/discussment/learning/panel/article/ArticleTextPanel.java

```diff
@@ -48,8 +47,9 @@ protected void onInitialize() {
         super.onInitialize();
 
         Label articleText = new Label ("articleText", new PropertyModel<String>(articleModel, "articleText"));
-
         add(articleText);
+
+        discussionModel = new PropertyModel(articleModel, "discussion");
```

Comment:
Why here? You can do this in the constructor instead of creating empty Model instance (which is then never used)